### ISSUE TITLE
Glossary button in deck editor

### ISSUE THREAD

**Issue by ShawnieBoy** [magarena/magarena#1108]
Just something that came up while making a deck - wasn't sure about an ability, but couldn't find a way to access the glossary from the deck editor screen

**Comment by lodici** [comment]
You can use the ``K`` (for Keywords) key to toggle the screen from anywhere. It was recently added but I forgot to update the [wiki page](https://github.com/magarena/magarena/wiki/Keyboard-shortcuts) so thanks for reminding me!

**Comment by ShawnieBoy** [comment]
Ahh, cool - didn't know about that, will have to remember.
Was wondering about an icon somewhere.

**Comment by lodici** [comment]
I agree, an icon in the top right corner of the screen would be useful too although I think it would need to be added to a number of other screens for consistency - explorer, decks, deck, etc.

**Comment by ShawnieBoy** [comment]
Yes, that's true - it's not something I check very often, but when I need it, it's frustrating to not be able to find it. Am I right in thinking that there was an icon at some point? Or was it in a different menu (I'm thinking esc menu)

**Comment by lodici** [comment]
Yes, previously you hit escape or an icon to display a menu from which you could then select the glossary screen but I got rid because it was implemented inconsistently such that on some screens escape would close the screen whilst on others it would open a menu. Now only the game screen displays a menu on escape, all other screens will close. Instead I have started looking at the iconized "slide" menu as used on the explorer screen for setting the style so could use something similar for other screen-related options.
